src/app/globals.css

```diff
@@ -35,5 +35,25 @@
 
 body {
   margin: 0;
+  background: #f2d43d;
   font-family: Atma, sans-serif;
 }
+
+h1 {
+  font-family: Atma, sans-serif;
```

Comment:
Maybe you can add a variable for the font-family? 🙂

Reply:
Thanks, I plan to set this font-family as global value. I hope that will do the job.